lib/sqitch-config.pod

```diff
@@ -484,6 +484,25 @@ by C<core.top_dir>.
 The file name extension on deploy, revert, and verify SQL scripts. Defaults to
 C<sql>.
 
+=item C<core.strict>
+
+A boolean value indicating whether or not to disallow certain
+potentially risky operations. Defaults to false.
+
+When enabled:
```

Comment:
Can you add tests for these conditions? I can't tell at a glance how some commands are allowed.

Reply:
Could you rephrase this request? I think the PR already adds tests for the behavior documented here.

Reply:
I see it now, sorry.